src/agent/agent.c

```diff
@@ -516,10 +516,12 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
         }
 
         value = cfg_get_value(agent->config, CFG_MANAGER_HOST);
-        if (value) {
-                if (!agent_set_host(agent, value)) {
-                        return false;
-                }
+        if (!value) {
```

Comment:
The default port is set in the [agent_new()](https://github.com/containers/hirte/blob/main/src/agent/agent.c#L323) and - if I understand it correctly - overridden if it is defined in any of the config options. For consistency, it should be the same for the host.